images/linux/scripts/installers/codeql-bundle.sh

```diff
@@ -4,22 +4,34 @@
 ##  Desc:  Install the CodeQL CLI Bundle to the toolcache.
 ################################################################################
 
-source $HELPER_SCRIPTS/install.sh
+source "$HELPER_SCRIPTS/install.sh"
 
 # Retrieve the name of the CodeQL bundle preferred by the Action (in the format codeql-bundle-YYYYMMDD).
-codeql_bundle_name="$(curl -sSL https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json | jq -r .bundleVersion)"
-# Convert the bundle name to a version number (0.0.0-YYYYMMDD).
-codeql_bundle_version="0.0.0-${codeql_bundle_name##*-}"
+defaults="$(curl -sSL https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json)"
+codeql_tag_name=${jq -r '.bundleVersion' <<< "${defaults}"}
```

Comment:
Should the outermost brackets be `$(...)` rather than `${...}`, since we're running a subshell command rather than interpolating a variable? Here and on the 3 lines below.

Reply:
Oh yes, thank you!

Reply:
Also don't forget the quotes around `"$(...)"`. Not essential here since the output doesn't have newlines, but good to be consistent.